lec01-introduction.Rmd

```diff
@@ -103,22 +107,18 @@ value of this sequence.
 mean(seq(0, 30))
 ```
 
-This is not as rich as the English language, which actually is something I am
-grateful for as an international student. There are few exceptions to the
-rules, and you rarely have to remember illogical grammar just because "that is
-how it is", I love it!
-
-And if you think it sounds complicated to remember all of these functions by
-heart, don't worry there are several places to get help. The one I use the most
+<!-- skipping this in lecture since it's covered in lec 2 anyways...
```

Comment:
But can you code without tabcompletion? I hit it automatically, so I just explain it here to not drive myself crazy trying to avoid it...